main.py

```
@@ -156,13 +184,73 @@ async def post_auth_token(
+        user["nickname"] = nickname
+    # 2. 비밀번호 변경이 요청된 경우 -- 회원가입 로직과 동일함.
+    if password:
+        validate_password(password)
+        user["hashed_password"] = hash_password(password)
+    # 3. 이미지 변경이 요청된 경우 -- 회원가입 로직과 동일함.
+    if profile_image:
+        if profile_image.content_type not in ALLOWED_TYPES:
+            raise HTTPException(415, "지원하지 않는 이미지 파일 형식입니다. ")
+        #TODO : 파일 저장 및 이미지 크기 확인 로직
+        await validate_image_size(profile_image)
+        profile_image_url = f"/uploads/{profile_image.filename}"
```

Ruff (0.14.11)

Local variable `profile_image_url` is assigned to but never used

Remove assignment to unused variable `profile_image_url`

(F841)